### PR TITLE
Fixed Markdown Links

### PR DESCRIPTION
**Pull request by LuciferUchiha**
Some markdown links were not written correctly for oracle and cisco, square brackets were missing or there were spaces between square and round brackets.

### CODE REVIEW

**Review by maertsen — CHANGES_REQUESTED** [review]
hi @LuciferUchiha, your help is much appreciated! Unfortunately your changes also include removals of unrelated lines. Can you fix those and/or re-open a PR that only includes the markdown-link-related changes? Thanks!

File: software/README.md [hunk 1]
```diff
@@ -743,6 +742,7 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 | Hexagon | Luciad Lightspeed | Unknown | Not vuln | The only risk is if Log4J was implemented outside of the default product install | [source](https://supportsi.hexagon.com/help/s/article/Security-Vulnerability-CVE-2021-44228-log4j-2)|
 | Hitachi Vantara | Pentaho | v8.3.x, v9.2.x | Not vuln | | [source](https://support.pentaho.com/hc/en-us/articles/4416229254541-log4j-2-zero-day-vulnerability-No-impact-to-supported-versions-of-Pentaho-)|
 | HostiFi | Unifi hosting | Unknown | Fix | Hosted Unifi solution | [source](https://twitter.com/hostifi_net/status/1440311322592231436) |
+| Huawei | All products | | Investigation | | [source](https://www.huawei.com/en/psirt/security-notices/huawei-sn-20211210-01-log4j2-en)|
```
Comment (maertsen) [inline software/README.md:745]: nit: can you correct the sorting on this one?

**Review by maertsen — CHANGES_REQUESTED** [review]
thanks! One small thing left.

File: software/README.md (hunk 1 above)
Comment (LuciferUchiha) [reply software/README.md]: Should be all good now :) I think I just got unlucky with the timing and got some changes from someone else.

Comment (maertsen) [reply software/README.md]: It's pretty busy :) Thanks!

Comment (LuciferUchiha) [reply software/README.md]: yeppp haha just noticed a few more oh well

**Pull request merged**